sample-app/src/components/UniversalResults.tsx

```diff
@@ -4,14 +4,24 @@ import { DecoratedAppliedFiltersConfig } from '../components/DecoratedAppliedFil
 import SectionHeader from "../sections/SectionHeader";
 import { SectionComponent } from "../models/sectionComponent";
 import { CardConfig } from '../models/cardComponent';
+import { useComposedCssClasses, CompositionMethod } from "../hooks/useComposedCssClasses";
 import classNames from "classnames";
-import '../sass/UniversalResults.scss';
+
+interface UniversalResultsCssClasses {
+  container?: string,
+  results___loading?: string
```

Comment:
In BEM, I know that the usual syntax is `[block]__[elem]`. Are triple underscores the convention when the block itself (`results`) is modified?

Reply:
I'm using the triple underscore to indicate the modifier since using dashes have some implications for CSS-Modules. See the [explanation here](https://github.com/yext/answers-headless-react/pull/67#discussion_r760471334)

Reply:
Got it! I thought you and Oshi had discussed this somewhere, but I couldn't find the comment.